models/lemonade/manifest.yaml

```diff
@@ -0,0 +1,31 @@
+version: 0.0.1
+type: plugin
+author: "lemonade-sdk"
```

Comment:
If you would like to submit the plugin to this repo, the author should be `langgenius`, otherwise you should submit a packaged plugin to this repo https://github.com/langgenius/dify-plugins/pulls